libcontainer/rootfs_linux.go

```diff
@@ -68,6 +74,7 @@ func prepareRootfs(pipe io.ReadWriter, iConfig *initConfig, mountFds []int) (err
 		cgroupns:        config.Namespaces.Contains(configs.NEWCGROUP),
 	}
 	setupDev := needsSetupDev(config)
+	//time.Sleep(30 * time.Second)
```

Comment:
Debug session leftover?

Reply:
yes, initial version, after moving helpers to x/sys/unix and runtime-spec, it would be clear.